src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php

```diff
@@ -34,12 +36,12 @@ function let(ObjectRepository $countryRepository, FormFactoryInterface $factory)
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Bundle\AddressingBundle\Form\EventListener\BuildAddressFormSubscriber');
+        $this->shouldHaveType(BuildAddressFormSubscriber::class);
```

Comment:
We can use these language construct even when class does not exist. PHP does not try to load the class, it only returns the FQCN.